tests/cephadm/test_package_validation.py

```diff
@@ -0,0 +1,162 @@
+import re
+
+from cli.utilities.containers import Container
+from cli.utilities.packages import Package, PackageError, ReposError, Rpm
+from cli.utilities.utils import get_custom_repo_url
+from utility.log import Log
+
+log = Log(__name__)
+
+
+YUM_REPO_DIR = "/etc/yum.repos.d"
+
+installed_packages = [
+    "chrony",
+    "lvm2",
+    "podman",
+]
+
+packages_to_install = [
+    "cephadm-ansible",
+]
+
+rpm_dependent_packages = [
+    "ansible-core",
+    "ansible-collection-ansible-posix",
+    "ansible-collection-community-general",
+]
+
+rpm_client_packages = [
+    "ceph-common",
+    "ceph-fuse",
+    "cephfs-top",
+]
+
+
+def validate_rpm_version(node, package_list, version_config):
+    """Validate installed rpm package version
+
+    Args:
+        node (ceph.ceph.CephNode): Node name
+        package_list (list): List of packages
+        version_config (dict): List of packages and package version
+    """
+    # Define regex to get version
+    rpm_regex = r"Version\s+:\s+(\d+\.\d+)"
```

Comment:
use `_regex`

Reply:
I think we can keep the name as rpm_regex itself. As if required for container we can distinct as conatiner_regex.

Reply:
This won't be added in the same method and method name itself have rpm